### PR TITLE
Enhancement: Removed asset manager

### PR DESCRIPTION
**Pull request by ins0**
This removes the asset manager. In the current state the asset manager don't provides real effort nor we use multi module public folders.

### CODE REVIEW

**Comment by localheinz** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,6 @@
         "ezyang/htmlpurifier": "4.6.*",
         "monolog/monolog": "~1.12",
         "php": "~5.5",
-        "rwoverdijk/assetmanager": "1.3.*",
```
Comment (localheinz) [inline composer.json]: @ins0 

You also need to run

```
$ composer update rwoverdijk/assetmanager
```

Comment (ins0) [inline composer.json]: done, some question - what is the point that the ``composer.lock`` file is in the  version control?

File: public/.gitignore [hunk 2]
```diff
@@ -1,4 +1 @@
-img
-css
-js
 favicon.ico
```
Comment (localheinz) [inline public/.gitignore]: @ins0 

I think this `.gitignore` can be removed, can't it?

Comment (ins0) [inline public/.gitignore]: yeah

File: composer.json (hunk 1 above)
Comment (Ocramius) [inline composer.json]: @ins0 the `composer.lock` MUST be under version control for deployments

File: public/.gitignore [hunk 3]
```diff
@@ -1,4 +0,0 @@
-img
```
Comment (Ocramius) [inline public/.gitignore]: Can we actually use bower for this stuff?

Comment (localheinz) [inline public/.gitignore]: @Ocramius 

:+1:, but out of scope of this PR, I think!

Comment (Ocramius) [inline public/.gitignore]: We are replacing with directly committed assets, heh

Comment (Ocramius) [inline public/.gitignore]: Nvm, we are just moving the assets. But yes, we should use bower here.

Comment (ins0) [inline public/.gitignore]: :+1:

**Comment by Ocramius** [comment]
:ship: :shipit:

**Comment by ins0** [comment]
thanks @localheinz and @Ocramius

**Comment by gianarb** [comment]
Thanks!!!! @ins0 :+1: :+1:

**Pull request merged**